tests/pipelines/stable_diffusion/test_stable_diffusion_adapter.py

```diff
@@ -186,8 +220,10 @@ def test_stable_diffusion_adapter_default_case(self):
         image = sd_pipe(**inputs).images
         image_slice = image[0, -3:, -3:, -1]
 
-        assert image.shape == (1, 64, 64, 3)
-        expected_slice = np.array([0.4858, 0.5500, 0.4278, 0.4669, 0.6184, 0.4322, 0.5010, 0.5033, 0.4746])
+        assert image.shape == (1, 128, 128, 3)
+        expected_slice = np.array(
+            [0.27978146, 0.36439905, 0.3206715, 0.29253614, 0.36390454, 0.3165658, 0.4384598, 0.43083128, 0.38120443]
+        )
```

Comment:
Note to reviewers:

You'll notice several `expected_slice` values changing in this PR. These values changed as a result of changes to `get_dummy_components(...)` in [6d4a060](https://github.com/huggingface/diffusers/pull/5435/commits/6d4a060a34415ec973a252944216f4fb8b9926cd). The changes to adapter.py **did not** require changes to `expected_slice`. I was careful to keep these changes in separate commits so that you can verify this, if desired.

Reply:
I still prefer to keep the default tests unchanged and instead add new tests. If possible, could we do that?

Reply:
Done. See my comment here: https://github.com/huggingface/diffusers/pull/5435#discussion_r1363960973

(My original comment here no longer applies.)